packages/react-dom-bindings/src/server/ReactDOMFloatServer.js

```diff
@@ -863,7 +863,7 @@ export function resourcesFromLink(props: Props): boolean {
     }
   }
   if (props.onLoad || props.onError) {
-    return false;
+    return true;
```

Comment:
this means we won't emit this element on the server. it's sort of a black hole where we created no resources but we also aren't going to render it as a component either. it will render as an insert on the client

Reply:
Maybe add this as a comment inline.

Reply:
I wonder if it would make sense to emit it as a kind of low priority preload though. It seems a bit late to wait for hydration to actually load it off the network.

Reply:
well as you pointed out before, we can't know if it is actually loadable. this is a generic pathway for links so this could be some strange rel like "foo".

Reply:
maybe known ones can still emit but also be inserted on the client. for instance preconnect and dns-prefetch and maybe even prefetch